contents/tutorials/single-page-app-pageviews.md

```diff
@@ -112,7 +112,77 @@ export default function RootLayout({ children }) {
 }
 ```
 
-Make sure to dynamically import the `PostHogPageView` component or the `useSearchParams` hook will deopt the entire app into client-side rendering. 
+Make sure to dynamically import the `PostHogPageView` component or the `useSearchParams` hook will deopt the entire app into client-side rendering.
+
+## Tracking pageviews in react-router-dom
+
+If you are using [react-router-dom](https://reactrouter.com/en/main), start by adding the `PostHogProvider` component in the `app` folder. Make sure to set `capture_pageview: false` because we will manually capture pageviews.
```

Comment:
```suggestion
If you are using [React Router](https://reactrouter.com/en/main) AKA `react-router-dom`, start by adding the `PostHogProvider` component in the `app` folder. Make sure to set `capture_pageview: false` because we will manually capture pageviews.
```